libcontainer/cgroups/systemd/common.go

```diff
@@ -11,6 +11,26 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+// Refer to https://github.com/coreos/go-systemd/blob/5a0db84d3dc459ccdc6ffcc44b1c452bf9f171cb/dbus/methods.go#L78-L101
+// or https://godoc.org/github.com/coreos/go-systemd/dbus#Conn.StartUnit
```

Comment:
these two are the same: one is the source code, another is godoc extracted from it.

And we only need "done", let's not overcomplicate things.

Reply:
Done

Reply:
In v1, shall we need to add this?
If yes, I'll add a const string `DbusJobDone`.